spras/config.py

```diff
@@ -254,7 +254,7 @@ def process_config(self, raw_config):
             raise ValueError("Evaluation analysis cannot run as gold standard data not provided. "
                              "Please set evaluation include to false or provide gold standard data.")
 
-        if 'aggregate_per_algorithm' not in self.ml_params:
-            self.analysis_include_ml_aggregate_algo = False
-        else:
+        if 'aggregate_per_algorithm' in self.ml_params and self.analysis_include_ml == True:
```

Comment:
There was a bug I found when I was running the code that 'aggregate_per_algorithm'  will still run when ML include is set to false.

Reply:
Now thinking about this behavior, would we want aggregate_per_algorithm to still run if ml include is set to false. Really the include is more of a ml_analysis_per_dataset.

Reply:
My interpretation of the config file values is that ml include True/False enables or disables everything. If it is False, then we do not care what `aggregate_per_algorithm` is set to because we are not running any of the ML analyses.

The current code looks like it matches that. However, `self.analysis_include_ml == True` can be simplified to `self.analysis_include_ml`.